pkg/api/v20220904/generate.go

```diff
@@ -3,4 +3,4 @@ package v20220904
 // Copyright (c) Microsoft Corporation.
 // Licensed under the Apache License 2.0.
 
-//go:generate go run ../../../hack/swagger github.com/Azure/ARO-RP/pkg/api/v20220904 ../../../swagger/redhatopenshift/resource-manager/Microsoft.RedHatOpenShift/stable/2022-09-04
+//go:generate go run ../../../hack/swagger github.com/Azure/ARO-RP/pkg/api/v20220904 ../../../swagger/redhatopenshift/resource-manager/Microsoft.RedHatOpenShift//openshiftclusters/stable/2022-09-04
```

Comment:
There is an extra slash before `openshiftclusters` that we could probably remove to be safe even though it didn't seem to cause problems yet.

Reply:
done, good catch